reactive/kubernetes_worker.py

```diff
@@ -1046,7 +1012,7 @@ def missing_kube_control():
                 hookenv.service_name()))
 
 
-@when('docker.ready')
+@when('endpoint.container-runtime.available')
 def fix_iptables_for_docker_1_13():
     """ Fix iptables FORWARD policy for Docker >=1.13
```

Comment:
Should we add something to check the runtime here and only do this on docker or does this issue hit containerd as well? Maybe update the comment if it is both.

Reply:
Yep, good point, I'll think about how to pass this information (inc. version) over the interface.

Reply:
I think this needs to move out of this charm and into the Docker sub.